ui_framework/dist/ui_framework.css

```diff
@@ -2239,7 +2239,7 @@ body {
   .kuiModalFooter > * + * {
     margin-left: 5px; }
 
-.kuiNoItems {
+.kuiTableInfo {
```

Comment:
I think you may have done a find/replace to change these, but this is a generated filed. So we'll need to update `ui_framework/components/index.scss` to remove the original component imports and add the new component import. We just need to change lines 21 through 53 to be this:

```scss
// Components
@import "action_item/index";
@import "badge/index";
@import "bar/index";
@import "button/index";
@import "card/index";
@import "collapse_button/index";
@import "column/index";
@import "event/index";
@import "form/index";
@import "form_layout/index";
@import "gallery/index";
@import "header_bar/index";
@import "icon/index";
@import "info_panel/index";
@import "link/index";
@import "local_nav/index";
@import "menu/index";
@import "menu_button/index";
@import "micro_button/index";
@import "modal/index";
@import "notice/index";
@import "panel/index";
@import "prompt_for_items/index";
@import "status_text/index";
@import "table/index";
@import "table_info/index";
@import "tabs/index";
@import "toggle_button/index";
@import "tool_bar/index";
@import "typography/index";
@import "vertical_rhythm/index";
@import "view/index";
```